clients/pkg/promtail/svc/service_windows.go

```diff
@@ -0,0 +1,44 @@
+//+build windows
+
+package svc
+
+import (
+	"time"
+
+	wsvc "golang.org/x/sys/windows/svc"
+)
+
+type svcWindows struct{}
+
+func init() {
+	interactive, err := wsvc.IsAnInteractiveSession()
```

Comment:
Apparently this is depracted in favor of [`IsWindowsService`](https://pkg.go.dev/golang.org/x/sys/windows/svc#IsWindowsService)